### PR TITLE
[FIX][Vms]Bugs of vms

### PR DESCRIPTION
**Pull request by lmgr0312**
This PR fixes different bugs: 

-The wrong path in some images of demo data
-The Tasks/Spare Parts when the preventive maintenances are triggered.
-The Create PO button in vms_order_line only appears when the order_line is an external part.
-The size of certains fonts in vms_kiosk_mode to prevent views with wrong design.

### CODE REVIEW

**Comment by lmgr0312** [comment]
@alan196 @ougc27 
Here we have some changes in the vms behavior... hope you have the chance to take a look around of this...

Regards!

**Comment by luistriana28** [comment]
👍

**Comment by ougc27** [comment]
Hello @lmgr0312 could you refactor this parameter please?

https://github.com/Jarsa-dev/vehicle-maintenance-system/blob/b75d76f3885261ecf42678b30cef75d5e72d22b3/vms/models/fleet_vehicle.py#L30-L31

Instead a system parameter could you create a field on a res.config.setting linked to the res.company please? :+1:

File: vms/models/vms_order_line.py [hunk 1]
```diff
@@ -115,6 +116,12 @@ def _compute_purchase_state(self):
                 rec.purchase_order_id.id and
                 rec.purchase_order_id.state == 'done')
 
+    @api.depends('spare_part_ids')
+    def _compute_create_purchase_order(self):
+        for rec in self:
+            rec.create_purchase_order = True if rec.spare_part_ids.filtered(
+                lambda x: x.external_spare_parts) else False
```
Comment (alan196) [inline vms/models/vms_order_line.py]: `bool(rec.spare_part_ids.filtered(lambda x: x.external_spare_parts))`

**Review by alan196 — CHANGES_REQUESTED** [review]

**Review by alan196 — APPROVED** [review]

**Pull request merged**